graspy/simulations/simulations.py

```diff
@@ -70,7 +70,7 @@ def sample_edges(P, directed=False, loops=False):
         return A - np.diag(np.diag(A))
 
 
-def er_np(n, p, directed=False, loops=False, wt=1, wtargs=None):
+def er_np(n, p, directed=False, loops=False, wt=1, wtargs=None, dc=None, dc_kws={}):
```

Comment:
add documentation for the new kwargs

Reply:
add tests for the new kwargs

Reply:
@bdpedigo I dont think there should be tests when the underlying function already does the tests.